htdocs/core/lib/process_template.lib.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
This file seems to be a service and not a library.
So it must be moved into dir
htdocs/core/ajax/process_template.php